docs/content/integrations/airflow/migrating-to-dagster.mdx

```diff
@@ -55,19 +55,31 @@ Even in the case that requires some infrastructure investment, `dagster-airflow`
 
 ---
 
-## Limitations
+## Deciding on persistent vs ephemeral airflow database
 
-Before kicking off a migration, note that there are some limitations to converting Airflow DAGs into Dagster jobs, assets, and schedules. The following features are currently unsupported:
+The dagster airflow migration tooling supports two methods for persisting airflow metadatabase state. By default it will us an ephemeral database that is scoped to every job run and thrown away as soon a job run terminates. You can also configure a persistent database that will be shared by all job runs. This tutorial uses the ephemeral database but you will want to use the persistent database option if you need the following features:
 
-- `retry-from-failure` in Dagster
-- Using `prev_execution_date` in Airflow-templated DAGs
+- `retry-from-failure` support in Dagster
+- passing airflow state between dag runs (ie xcoms)
 - `SubDAGOperators`
-- Airflow Datasets
 - Airflow Pools
 
-Most of the current limitations are [due to a locally-scoped, ephemeral Airflow database being created for each run](https://github.com/dagster-io/dagster/issues/12209). If your DAGs implicitly rely on Airflow database state across `DagRuns`, then the current migration tooling won't work for you.
+If you have complex airflow DAGs (cross-dag state sharing, pools) or the ability to retry-from-failure you will need to have a persistent database when making your migration.
```

Comment:
Is `retry-from-failure` the name of a feature or a literal value you'd see in code?

Reply:
neither, it's an attempt to describe when a multi-op job has failed and you want to retry from the op that failed, not retry/rerun the entire job and ops that had succeeded